travis.cfg

```diff
@@ -1,12 +0,0 @@
-[buildout]
-extends = buildout.cfg
-parts += createcoverage
-
-[code-analysis]
-recipe = plone.recipe.codeanalysis
-pre-commit-hook = False
-return-status-codes = True
```

Comment:
Code analysis is already in buildout.cfg, just with a different configuration. This discussion really gets out of hand, we are discussing 10 issues at the same time.